src/grg_sphinx_theme/team.py

```diff
@@ -85,14 +94,33 @@ def get_teams(github_project:str, github_teams:list, contributors_details:list =
             }
             teams_data.append(team_data)
         else:
-            logger.info("Unable to fetch team data for team: {0}. Check your API key or the name of the team!".format(team))
+            logger.info(f"Unable to fetch team data for team: {team}. "
+                        + "Check your API key or the name of the team!")
     return teams_data
 
-def get_contributors(github_project:str, github_repo:str, contributors_details:list = None):
+
+def get_contributors(github_project: str, github_repo: str,
+                     contributors_details: list = None):
     """Fetch list of contributors from github."""
+    page = 1
+    contributors = []
+
+    while (True):
+        url = (BASE_URL +
+               f"repos/{github_project}/{github_repo}/" +
+               f"contributors?per_page=100&page={page}")
+
+        contributor_page = get_json_from_url(url)
+        for con in contributor_page:
+            print(con['login'])
+
+        contributors += contributor_page
+
+        if len(contributor_page) < 100:
```

Comment:
just in case,of a bug we do not see, can you do `len(contributor_page) < 100 or page > 100:`